### PR TITLE
Use `onChange` in select example

### PR DESCRIPTION
**Pull request by andrewiggins**
Update the `select` example to use the `onChange` event which has better browser support (namely IE & old-Edge)

### CODE REVIEW

File: content/en/guide/v10/forms.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ class MySelect extends Component {
   render(_, { value }) {
     return (
       <form onSubmit={this.onSubmit}>
-        <select value={value} onInput={this.onInput}>
+        <select value={value} onChange={this.onInput}>
```
Comment (andrewiggins) [inline content/en/guide/v10/forms.md]: Should I change the name of the method as well?

Comment (developit) [reply content/en/guide/v10/forms.md]: yeah, lets.

Comment (andrewiggins) [reply content/en/guide/v10/forms.md]: Done

**Comment by andrewiggins** [comment]
I also updated the example to display the the current value in the alert box so its easier to verify that the sample works in the REPL

**Pull request merged**